index.rst

```diff
@@ -536,7 +597,12 @@ After the initial test it seemed like the Jupyter notebooks from the nublado pla
 During the tests of the features introduced above it was possible to demonstrate that the single process mode is now capable of joining a domain and communicating with the CSCs in the system.
 In particular, the feature that seems to allow the use of single process mode is the introduction of the :ref:`DDS data partitioning <Implementing-a-new-DDS-data-partitioning-schema>`.
 
-This considerably simplify handling the nublado configuration as the containers can continue to be hosted in public repositories, there is no need to configure them to use shared memory and to worry about users notebooks crashing and corrupting the daemons they are attached to, potentially affecting other systems.
+Although single process from notebooks **can** work with a large scale system deployed with shared memory, we did noticed some small issues.
+Probably the most critical one is that creating a salobj Remote soon after creating a Domain will, in most cases, cause a HRTO_ problem.
+After inspecting the DDS logs it was possible to track this issue to trying to start DDS readers while the domain was still aligning with the other systems.
```

Comment:
Is this where the 10 second delay after creating a domain recommendation comes from while we have nublado in single process mode?

Reply:
Exactly...